lib/app_perf_rpm/introspector.rb

```diff
@@ -0,0 +1,30 @@
+module AppPerfRpm 
+  class Introspector
+
+    VALID_RUNNERS = %w(Puma Unicorn Sidekiq Sinatra Resque Webbrick)
```

Comment:
Does Thin register as webbrick? What about passenger? Should we add DelayedJob in here as well? I also use a custom one called Emque. https://github.com/emque/emque-producing and https://github.com/emque/emque-consuming